### DISCUSSION TITLE
Generation of Summarized Experiment from ArchR Project

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1734] [General]
Hi Ryan,

 What is the best way to convert the ArchR project to a Summarized Experiment and/or a Ranged Summarized Experiment?

Thanks!

**Comment by rcorces** [comment]
You cant convert and ArchRProject to a SummarizedExperiment - there is far too much information in an ArchRProject to be stored in a SummarizedExperiment. But I believe this will do what you are looking for (for a single matrix)
https://www.archrproject.com/reference/getGroupSE.html

**Reply by sylestiel** [reply]
Would you be able to advice if I could use this output to run FigR?

**Reply by rcorces** [reply]
I dont see why not but I've never used that tool
